sound/soc/sof/utils.c

```diff
@@ -123,12 +123,18 @@ static void sof_get_windows(struct snd_sof_dev *sdev)
 	u32 outbox_size = 0;
 	u32 stream_size = 0;
 	u32 inbox_size = 0;
+	int bar;
 	int i;
 
 	if (!sdev->info_window) {
 		dev_err(sdev->dev, "error: have no window info\n");
 		return;
 	}
+	bar = snd_sof_dsp_get_bar_index(sdev, SOF_FW_BLK_TYPE_SRAM);
+	if (bar < 0) {
+		dev_err(sdev->dev, "error: have no bar mapping\n");
+		return;
+	}
```

Comment:
@dbaluta I think this should be squashed with the previous patch, I mean using the bar instead of BYT_DSP_BAR in the common sof_fw_ready() implementation.

Reply:
@ranj063 you are right, the compilation even fails. Working to fix this.